early-init.el

```diff
@@ -0,0 +1,13 @@
+;; Before Emacs 27, the init file was responsible for initializing the package
```

Comment:
You probably have to add the copyright notice at the top of this file:

```
;;; early-init.el --- Spacemacs Initialization File
;;
;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
;;
;; Author: Your details here <=============================================================
;; URL: https://github.com/syl20bnr/spacemacs
;;
;; This file is not part of GNU Emacs.
;;
;;; License: GPLv3
```

See the `init.el` file for an example.

Reply:
Good call.  Fixed.